openeo/extra/job_management.py

```diff
@@ -59,11 +59,32 @@ def read(self) -> pd.DataFrame:
     def persist(self, df: pd.DataFrame):
         """
         Store job data to the database.
+        The provided dataframe may contain partial information, which is merged into the larger database.
 
         :param df: job data to store.
         """
         ...
 
+    @abc.abstractmethod
+    def has_unfinished_jobs(self) -> bool:
+        """
+        Check if there are still unfinished jobs in the database.
+
+        :return: True if there are unfinished jobs, False otherwise.
+        """
+        ...
+
+    @abc.abstractmethod
+    def get_by_status(self, include, max=None) -> pd.DataFrame:
```

Comment:
```suggestion
    def get_by_status(self, include: List[str], max=None) -> pd.DataFrame:
```

Reply:
I'm also a bit confused by name "include", is that because you anticipate to add an "exclude" list later?

Reply:
include -> statuses

Reply:
done